### ISSUE TITLE
Dark Theme doesn't work

### ISSUE THREAD

**Issue by sirinath** [insipx/Ergo#50]
The dark theme in the example site does not work.

**Comment by JoePricey** [comment]
You're right. But it works on my own deployed site and also running the test server locally.

Guessing it's something wrong with this specific deployed instance, and not the code itself...

```
Uncaught ReferenceError: main is not defined
    onload https://ergo.liquidthink.net/:1
[ergo.liquidthink.net:1:1](https://ergo.liquidthink.net/)
    onload https://ergo.liquidthink.net/:1
```

**Comment by JoePricey** [comment]
@insipx Think there's something up with your demo site? Can't reproduce on any other user of this project.

**Comment by insipx** [comment]
I'll have to probably re-deploy the demo site at some point, probably something up with netlify
